packages/esm-patient-immunizations-app/src/immunizations/delete-immunization.modal.test.tsx

```diff
@@ -4,11 +4,13 @@ import { render, screen } from '@testing-library/react';
 import { showSnackbar, useConfig } from '@openmrs/esm-framework';
 import { mockPatient } from 'tools';
 import { deletePatientImmunization } from '../hooks/useImmunizations';
+
```

Comment:
Please don't commit irrelevant changes in a PR. These added lines are both unnecessary and not related to the core of what this PR is doing.